include/srtp_priv.h

```diff
@@ -107,21 +135,16 @@ typedef enum direction_t {
 
 typedef struct srtp_stream_ctx_t_ {
   uint32_t   ssrc;
-  srtp_cipher_t  *rtp_cipher;
-  srtp_cipher_t  *rtp_xtn_hdr_cipher;
-  srtp_auth_t    *rtp_auth;
+  srtp_session_keys_t *session_keys;
+  unsigned int num_master_keys;
   srtp_rdbx_t     rtp_rdbx;
   srtp_sec_serv_t rtp_services;
-  srtp_cipher_t  *rtcp_cipher;
-  srtp_auth_t    *rtcp_auth;
   srtp_rdb_t      rtcp_rdb;
   srtp_sec_serv_t rtcp_services;
   srtp_key_limit_ctx_t *limit;
```

Comment:
I think I have the changes.  Is there a specific test for the limit code?  I ran all the test and everything passes.  I am going to push the changes but if you know of a specific limit test that is being run that will make me have greater confidence in the quick change I just made.